addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3312,34 +3312,6 @@ public Action:MakeBoss(Handle:timer, any:boss)
 		}
 	}
 
-	entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_wearable_demoshield"))!=-1)
-	{
-		if(IsBoss(GetEntPropEnt(entity, Prop_Send, "m_hOwnerEntity")))
-		{
-			TF2_RemoveWearable(client, entity);
-		}
-	}
-
-	entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_usableitem"))!=-1)
```

Comment:
They technically aren't connected to a physical entity, but they still have a trackable item index, but to block it you need to kill the animation entity and replace it with another, otherwise bad things occur

I would share some code, but it's private for the community I work for